helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/api/V1HelpDeskController.java

```diff
@@ -138,26 +137,31 @@ private MakerCheckerResponse buildMakerCheckerResponse(SymUser agentUser,
     return makerCheckerResponse;
   }
 
-  private void sendAcceptMarkerChekerMessages(MakerCheckerMessageDetail detail) {
+  private void sendApprovedMakerChekerMessage(MakerCheckerMessageDetail detail) {
     AttachmentMakerCheckerMessage checkerMessage = new AttachmentMakerCheckerMessage();
     checkerMessage.setAttachmentId(detail.getAttachmentId());
     checkerMessage.setGroupId(detail.getGroupId());
     checkerMessage.setMessageId(detail.getMessageId());
-    checkerMessage.setStreamId(
-        Base64.encodeBase64URLSafeString(Base64.decodeBase64(detail.getStreamId())));
+    checkerMessage.setStreamId(Base64.encodeBase64URLSafeString(Base64.decodeBase64(detail.getStreamId())));
     checkerMessage.setProxyToStreamIds(detail.getProxyToStreamIds());
     checkerMessage.setTimeStamp(detail.getTimeStamp());
     checkerMessage.setType(detail.getType());
 
-    Set<SymMessage> symMessages = agentMakerCheckerService.getAcceptMessages(checkerMessage);
-
-    AiSessionKey aiSessionKey = helpDeskAi.getSessionKey(detail.getUserId(), detail.getStreamId());
+    Set<SymMessage> symMessages = agentMakerCheckerService.getApprovedMakercheckerMessage(checkerMessage);
 
     for (SymMessage symMessage : symMessages) {
```

Comment:
You might use Java 8 streams with map and forEach to make the code more readable.